Bottom: b8538c94bdd141fcd437e46be51ab9f5bf5c84f8
Top:    fbe5c02ccf6099a9713fe1ce88cac01f66e9b39c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:57:38 +0000

Refresh of hid-batch-add-some-missing-hea

---

diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index 2c2b926..558f9fb 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -12,6 +12,8 @@
 #include "global.h"
 #include "hid.h"
 #include "data.h"
+#include "misc.h"
+#include "hid.h"
 
 #include "hid/common/actions.h"
 #include "hid/common/hidint.h"
